### ISSUE TITLE
[Bug Report] run_scans.py does not discovers vfatConfig if using trim option

### ISSUE THREAD

**Issue by bdorney** [cms-gem-daq-project/vfatqc-python-scripts#259]
## Brief summary of issue

Title.

Otherwise trimming will either be done at ARM DAC = 100 or apply a single VFAT config to all links (bad).

### Types of issue

- [ ] Bug report (report an issue with the code)
- [x] Feature request (request for change which adds functionality)

## Expected Behavior

Like it discovers calibration files it should discover VFAT Config.

## Current Behavior

It does not discover VFAT config.

## Context (for feature requests)

Trimming at wrong ARM DAC vs. used ARM DAC yields bad results, example:

http://cmsonline.cern.ch/cms-elog/1084287

## Your Environment

* Version used:
* Shell used:

**Comment by bdorney** [comment]
# Solution to the Bug

The following lines:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/54711c0191d20c2c3cfa026b56d01eb6a393f07c/run_scans.py#L331-L332

Should be changed to something like:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/54711c0191d20c2c3cfa026b56d01eb6a393f07c/run_scans.py#L282-L303

Simple implementation:

```python
        # Check to see if a vfatConfig exists
        vfatConfigFile = "{0}/configs/vfatConfig_{1}.txt".format(dataPath,chamber_config[ohKey])
        if os.path.isfile(vfatConfigFile):
            cmd.append("--vfatConfig={}".format(vfatConfigFile))
```

Then I would suggest to drop the `vfatConfig` argument for this sub command.

**Comment by bdorney** [comment]
Addressed by #260
